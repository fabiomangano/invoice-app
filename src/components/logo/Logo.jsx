import React from 'react';
import { ReactComponent as LogoIcon } from '../../assets/logo.svg';
import './Logo.style.scss';

const Logo = () => (
  <div className="logo">
    <div className="logo__top-box" />
    <div className="logo__bottom-box" />
    <LogoIcon className="logo__icon" />
  </div>
);

export default Logo;
