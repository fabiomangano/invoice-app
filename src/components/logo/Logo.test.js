import React from 'react';
import Renderer from 'react-test-renderer';
import Logo from './Logo';

describe('Logo', () => {
  it('Should render correctly the main app logo', () => {
    let tree;
    Renderer.act(() => {
      tree = Renderer.create(<Logo />);
    });
    expect(tree.toJSON()).toMatchSnapshot();
  });
});
