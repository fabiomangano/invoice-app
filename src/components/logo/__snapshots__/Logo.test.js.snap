// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Should render correctly the main app logo 1`] = `
<div
  className="logo"
>
  <div
    className="logo__top-box"
  />
  <div
    className="logo__bottom-box"
  />
  <IconMock
    className="logo__icon"
  />
</div>
`;
